requirements.txt

```diff
@@ -7,13 +7,14 @@ botocore==1.13.19
 cffi==1.11.5
 cryptography==2.3.1
 dataclasses<=0.6
-elasticsearch>=5.0.0,<6.0.0
-elasticsearch-dsl>=5.0.0,<6.0.0
+elasticsearch==6.8.1
+elasticsearch-dsl==6.4.0
 jsonobject==0.9.9
 Faker==0.8.11
 furl==2.0.0
 future==0.16.0
 hca==7.0.1
+httplib2==0.17.2
```

Comment:
Is this a transitive dependency?

Reply:
That is correct, it seem to have played a role in the test that was failing.

Reply:
This was removed, since it was not actually the root of the problem. The issues has been addressed.